### PR TITLE
Signal handlers, clear terminal title on shutdown.

### PR DESCRIPTION
**Pull request by PJB3005**
The signal handling code only gets compiled under Mono. The `Console.CancelKeyPress` code always does though.

### CODE REVIEW

File: SS14.Server.Services/SS14Server.cs [hunk 1]
```diff
@@ -47,6 +47,8 @@ public void Restart()
             instance.Restart();
         }
 
+        public void Shutdown(string reason=null) => instance.Shutdown(reason);
```
Comment (ComicIronic) [inline SS14.Server.Services/SS14Server.cs]: Don't break consistency, make this a full method body.

**Comment by psykzz** [comment]
Why does the title need to be cleared?

**Comment by PJB3005** [comment]
@psykzz if we don't clear it explicitly, it stays after the process exits:

afbeelding

See here my terminal tab title *and* window title (due to being the selected tab) being the title from the non-running server.

**Comment by psykzz** [comment]
Makes sense, thanks.

**Comment by psykzz** [comment]
@PJB3005 conflicts

File: SS14.Server/Signals.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            UnixSignal[] signals = new UnixSignal[] {
+                new UnixSignal (Mono.Unix.Native.Signum.SIGTERM),
+                new UnixSignal (Mono.Unix.Native.Signum.SIGINT)
+            };
+
+            SignalThread = new Thread(() =>
+            {
+                while (true)
+                {
+                    Console.WriteLine("hrm?");
```
Comment (PJB3005) [inline SS14.Server/Signals.cs]: Wait shit I forgot to remove this

**Comment by ZoldorfTheWizard** [comment]
@PJB3005  SS14.Server.Services/ServerConsole/ConsoleManager.cs conflicts

**Pull request merged**